### PR TITLE
Label Props

### PR DESCRIPTION
**Pull request by allcentury**
Added missing labels found here:
https://github.com/grommet/grommet/blob/master/src/js/components/Label.js#L44-L51

Specifically:

- margin
- size
- announce

#### What does this PR do?

Adds missing label props

#### Where should the reviewer start?

See issue #225 

#### What testing has been done on this PR?

Reviewed via `gulp dev`

#### How should this be manually tested?

Review via `gulp dev`

#### Any background context you want to provide?

Issue #225 

#### What are the relevant issues?

Issue #225 

#### Screenshots (if appropriate)

### CODE REVIEW

File: src/docs/components/LabelDoc.js [hunk 1]
```diff
@@ -35,6 +35,12 @@ export default class LabelDoc extends Component {
             <dt><code>uppercase   true|false</code></dt>
             <dd>Convert the label to uppercase.
               Defaults to <code>false</code>.</dd>
+            <dt><code>margin    none|small|medium|large</code></dt>
+            <dd>Margin size</dd>
+            <dt><code>size    small|medium</code></dt>
+            <dd>Size of label defaults to <code>medium</code></dd>
+            <dt><code>announce    true|false</code></dt>
+            <dd></dd>
```
Comment (allcentury) [inline src/docs/components/LabelDoc.js]: I'm unsure what the description should be for `announce`.

Comment (alansouzati) [reply src/docs/components/LabelDoc.js]: I can add that description myself. Thanks for adding this

**Review by RyanCCollins — APPROVED** [review]
Thanks so much for the PR to fix this.  It is very much appreciated @allcentury!

@alansouzati I hope you don't mind me jumping in on this.   Please feel free to make any changes you want to the description of the announce prop.  Otherwise, I think this is ready to be merged.

**Comment by alansouzati** [comment]
Perfect thank you guys!!! 🎉

**Pull request merged**